contentcuration/contentcuration/views/files.py

```diff
@@ -55,6 +55,7 @@ def file_create(request):
     if request.method != 'POST':
         return HttpResponseBadRequest("Only POST requests are allowed on this endpoint.")
 
+    # import ipdb; ipdb.set_trace()
```

Comment:
Also delete!

Reply:
This breakpoint is triggered before the mysterious error!